### PR TITLE
Add Logo and Update Favicon

### PR DESCRIPTION
**Pull request by sprankhub**

### CODE REVIEW

File: src/components/Header.astro [hunk 1]
```diff
@@ -2,32 +2,73 @@
...
-  </a>
+      <span
+        class="
+            block
+            text-center
+          "
+      >
+        The Magento OS Community Alliance
+      </span>
+    </a>;
```
Comment (wigman) [inline src/components/Header.astro:73]: 👀 I spy a little extra semi-colon

**Review by wigman — CHANGES_REQUESTED** [review]
we're almost there :D thanks for making the changes! 🙌

File: src/components/Header.astro (hunk 1 above)
Comment (sprankhub) [reply src/components/Header.astro:73]: True. That was part of the file you sent me and I thought it was something Astro-specific (never worked with Astro before) :joy:  If it is not, happy to remove it. Sec...

Comment (wigman) [reply src/components/Header.astro]: Sorry 🙈

**Pull request merged**